### PR TITLE
add Erdil to ember-users

### PR DESCRIPTION
**Pull request by sly7-7**

### CODE REVIEW

File: data/users.yml [hunk 1]
```diff
@@ -890,6 +890,10 @@
   image: g5_x2.png
   alt: G5
   use: "G5 uses Ember.js for our content management system and various inventory search applications."
+- url: https://www.erdil.fr
+  image: erdil.png
+  alt: Erdil
+  use: "Erdil uses ember.js for it's website and all of it's applications'"
```
Comment (locks) [inline data/users.yml]: Thank you for using Ember :D Could you do some small tweaks?

`ember.js` -> `Ember.js`
`it's website` -> `its website`
`it's applications'` -> `its applications.`

Comment (sly7-7) [inline data/users.yml]: Stupid French guy 😄

**Comment by wifelette** [comment]
Email received, this is good to merge once @locks is good!

**Comment by locks** [comment]
Thank you!

**Pull request merged**